### PR TITLE
Bud 94 date slider

### PR DESCRIPTION
**Pull request by JadaTBK**
The date slider is a thing now. It sends the dates to Firestore too. I just made the visual style try to be similar to default iOS stuff. No unit tests yet.

### CODE REVIEW

File: Buddies/CreateActivity/CreateActivityVC.swift [hunk 1]
```diff
@@ -58,6 +59,8 @@ class CreateActivityVC: UITableViewController, UITextViewDelegate, UITextFieldDe
                 description: description,
                 location: GeoPoint(latitude: chosenLocation.latitude,
                                    longitude: chosenLocation.longitude),
+                startTime: getSliderDate(sliderValue: dateSlider.minValue),
```
Comment (noahtallen) [inline Buddies/CreateActivity/CreateActivityVC.swift]: Could you use `start_time` and `end_time` instead? That's what we have in the [data model doc](https://docs.google.com/document/d/1GhrXckvtJGYOKy8wnEAVBvP1aOvDOrLmKcD9wpfECtg/edit) ;)

**Review by noahtallen — CHANGES_REQUESTED** [review]
Wow this is REALLY cool! I like this a lot. I just made a small comment, but other than that, this is good to go!

File: Buddies/CreateActivity/CreateActivityVC.swift (hunk 1 above)
Comment (JadaTBK) [reply Buddies/CreateActivity/CreateActivityVC.swift]: Changed! (I used 'start_time' and 'end_time' when saving in Firestore but I changed the parameter names too. :3

**Review by noahtallen — APPROVED** [review]

File: Buddies/CreateActivity/CreateActivityVC.swift (hunk 1 above)
Comment (noahtallen) [reply Buddies/CreateActivity/CreateActivityVC.swift]: Oops I'm sorry! I thought what I commented on was the actual firebase code.. 🤦‍♂️

**Pull request merged**